boards/arm/nucleo_wb55rg/Kconfig.defconfig

```diff
@@ -44,34 +34,21 @@ config I2C_3
 
 endif # I2C
 
-if SPI
-
 config SPI_1
 	default y
-
-endif # SPI
-
-if ADC
+	depends on SPI
 
 config ADC_1
 	default y
-
-endif # ADC
-
-if BT_DEBUG_MONITOR
+	depends on ADC
 
 config LPUART_1
```

Comment:
This symbol was defined under 2 different if, and now is defined twice with 2 different "depends on".
Is that strictly equivalent ?

Reply:
Yup, equivalent. `if` just adds a `depends on` to all symbols within it. `depends on` is the only thing that "really" exists. `if` is just sugar.